app/routes/agenda/agendaitems/agendaitem/index.js

```diff
@@ -26,8 +26,7 @@ export default class DetailAgendaitemAgendaitemsAgendaRoute extends Route {
     if (this.subcase) {
       this.submitter = await this.subcase.requestedBy;
       this.meeting = await this.subcase.requestedForMeeting;
-      const case_ = await this.subcase.case;
-      await case_.governmentAreas;
+      await this.subcase.governmentAreas;
```

Comment:
Is this preloading necessary?  The data will automatically be fetched by the template where `this.subcase.governmentAreas` gets called.

Reply:
it seems needed.
Broken without.
![image](https://user-images.githubusercontent.com/22245223/185935289-a19e8e65-c7e3-4dbf-92c7-1d17ab4053b6.png)

I think

Reply:
In the util, when not preloading the proxy is not resolved as expected, resulting in 0 areas
this line `yield this.args.governmentAreas?.toArray();`

If I change it to this:
```
const governmentAreasPromise = yield this.args.governmentAreas;
const governmentAreas = yield governmentAreasPromise?.toArray();
```
Then it does working without preloading (still testing if it's reliable in all cases)

Reply:
Will try to remove the promises though, not very robust.